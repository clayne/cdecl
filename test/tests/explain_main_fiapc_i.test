cdecl @ @ @ explain int main( int, char *argv[] ) @ 0
