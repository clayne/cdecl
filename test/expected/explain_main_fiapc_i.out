declare main as function (int, argv as array of pointer to char) returning int
